subcommittees/maintainers/release-process.md

```diff
@@ -0,0 +1,50 @@
+# Release Process and Schedule
+
+We have decided to use a **6-month release cycle** for releases, which
+will be named by the year and month they happen, e.g., `2016.05`.
+
+## Overview
+
+1. Each lesson will continue live in the `gh-pages` branch of its own repository.
+2. When a release has to be made, the *lesson maintainer* (or maintainers) will create a branch named after the release, e.g., `2016.05`.
```

Comment:
On Sun, May 15, 2016 at 07:03:06AM -0700, Greg Wilson wrote:
> > +2. When a release has to be made, the *lesson maintainer* (or
> > maintainers) will create a branch named after the release, e.g.,
> > `2016.05`.
> 
> I believe this was discussed by the maintainers before the vote.

I haven't found branch vs. tag discussion in the recent maintainers@
thread, with the closest thing probably being [1] using ‘versiontag’.
The message kicking off the discussion is [2], and the linked
procedure uses branches [3].  There's a later post sumarizing thread
discussion [4], but still no branch vs. tag discussion.  I'll look for
earlier threads to see if I can turn up the branch vs. tag arguments.

I haven't found an actual vote or results yet either ;).  Will post
back once I do.

[1]: http://lists.software-carpentry.org/pipermail/maintainers/2016-April/000230.html
[2]: http://lists.software-carpentry.org/pipermail/maintainers/2016-April/000222.html
[3]: https://github.com/twitwi/swc-meta/blame/1e1c7fc7e02a197f3b939b10a16be1589d068bae/release-process.md#L15
[4]: http://lists.software-carpentry.org/pipermail/maintainers/2016-April/000245.html